### DISCUSSION TITLE
I would like to see a new feature, please add? ( If possible )

### DISCUSSION THREAD

**Discussion by n0bzeR** [LukeZGD/Legacy-iOS-Kit#437] [Ideas]
**Hello; good day.**
 If possible, I would like to **suggest adding a rollback option to iOS 7 - for iPhone 5s** (A7 , 64 Bit)
I found this site, it says that it is **possible to roll back to 7 iOS - 5S !!! use: iOS 7 Tethered Downgrade**
**Here is the site itself** - http://ios7.iarchive.app/downgrade/

If possible, please add a rollback to version 7?
I really want the old, native firmware! Now 10.3.3 is installed!
I have a Hackintosh Sierra (10.12.6) and Linux
It would be very cool if you added a rollback to this version!
;

" Jailbroken A7 device on latest iOS with openssh installed (any compatible jailbreak should work)

The target firmware (iOS 7.x) which can be found [here](https://ipsw.me/)

You will also need to download an iOS 8.0 IPSW to make an SSH ramdisk

NOTE: Due to Apple messing up their servers, you may need to download the IPSW using http. To do this, remove "secure-" from the URL and replace "https" with "http"

SHSH(2) (apticket) files:
SHSH signatures are required in order to properly build img4 files that the BootROM will accept from pwned DFU mode. Unless you have them already, you can save them with tsschecker (or other public tools.)

NOTE: SHSH version does not have to match target iOS version. "
****
I really **liked your program**!
  It's very easy to use, I'm glad!!
Thank you for that!!
  It would be great if you could add this feature!
**Thanks for reading!**

**Comment by LukeZGD** [comment]
There actually is a script for this in the works, here: https://github.com/y08wilm/a7-ios7-downgrader

**Reply by n0bzeR** [reply]
Thank you very much!
  I will definitely have to try this!
You'll have to install Hackintosh High Sierra 10.13
(Well, it's worth it! I really want the old iOS!)
Thanks again! We only managed to find a way to roll back to 10.3.3
  Thanks to you for sharing the link, I’ll set it to 7!

**Comment by n0bzeR** [comment]
> There actually is a script for this in the works, here: https://github.com/y08wilm/a7-ios7-downgrader

... Installed as recommended OS: 10.13.6

but I can’t install dependencies for the script
Rust, GCC
It says something like - outdated system, 10.13.6 (does not allow installation via home brew)

Well, I’ll try to install Catalina Mac OS, I hope it will install all the necessary dependencies +_+
Thanks again! I really want to install iOS 7
